src/runtime/data/kata-collect-data.sh

```diff
@@ -844,6 +844,49 @@ show_latest_kata_version()
 	fi
 }
 
+show_third_party_tools()
+{
+	not_found="Tool not found"
+	start_section "Third Party Tools"
+
+	conmon_version="$(conmon --version | awk -v FS="(^conmon version)|\n" '{print $2}' | xargs || true)"
+	[ -z $conmon_version ] && conmon_version=$not_found
+	msg "conmon version: $conmon_version"
+
+	crio_version="$(crio --version | awk -v FS="(^Version:)|\n" '{print $2}' | xargs || true)"
+	[ -z $crio_version ] && crio_version=$not_found
+	msg "cri-o version: $crio_version"
+
+	# Not in vagrant
+	containerd_version="$(containerd --version | awk -F ' ' 'NR==1{print $3}' | xargs || true)"
+	[ -z $containerd_version ] && $containerd_version=$not_found
+	msg "containerd version: $containerd_version"
+
+	critools_version="$(crictl --version | awk -v FS="(^crictl version)|\n" '{print $2}' | xargs || true)"
+	[ -z $critools_version ] && critools_version=$not_found
+	msg "critools version: $critools_version"
+
+	# Not in vagrant
+	gperf_version="$(gperf --version | awk -v FS="(GNU gperf)|\n" '{print $2}' | xargs || true)"
+	[ -z $gperf_version ] && gperf_version=$not_found
+	msg "gperf version: $gperf_version"
+
+	kubernetes_version="$(kubectl version --output=json | jq -r '.clientVersion.gitVersion' || true)"
```

Comment:
kubectl will find a kubeconfig file on some paths and/or the `KUBECONFIG` variable. So it can fail in case the config file is not found.

Reply:
Do you mean that we should do better failure handling, or that there should be a better way to do this?

I'm going to redirect all the standard errors for the comment above.